package/boot/uboot-envtools/files/mediatek_filogic

```diff
@@ -56,6 +56,7 @@ cmcc,rax3000m)
 	;;
 cetron,ct3003|\
 netgear,wax220|\
+zbtlink,zbt-z8103ax|\
```

Comment:
Please maintain alphabetic order, ie. put z8103ax after z8102ax.

Reply:
Yes! Sorry, I was having a hard time rebasing. Thank you✌️🤞